src/components/marketplace/buy_sell.tsx

```diff
@@ -135,14 +135,27 @@ const TokenContainer = styled.div`
     z-index: 12;
 `;
 
-const TokenText = styled.span`
+const tokenTextStyle = `
     color: #333;
     font-size: 14px;
     font-weight: normal;
     line-height: 21px;
     text-align: right;
 `;
 
+const TokenText = styled.span`
+    ${tokenTextStyle}
+`;
+
+const TokenTextUppercase = styled.span`
+    ${tokenTextStyle}
+    text-transform: uppercase;
+`;
```

Comment:
This can be written as:

```javascript
const TokenText = styled.span`
    color: #333;
    font-size: 14px;
    font-weight: normal;
    line-height: 21px;
    text-align: right;
`;

const TokenTextUppercase = styled(TokenText)`
    text-transform: uppercase;
`;
```

I haven't used `styled-components` enough to assure that this is more idiomatic, but it's shorter and (IMO) clearer.

I'm going to approve it anyway because it's not reason enough to block the PR.

Reply:
Done!